src/pages/Setting.tsx

```diff
@@ -1,43 +1,78 @@
 import styled from '@emotion/styled';
-import { GoogleOAuthProvider } from '@react-oauth/google';
+import { useState } from 'react';
 
 import useGetUserInfo from '@/apis/user/query';
+import NavBar from '@/components/common/NavBar';
 import LoadingSpinner from '@/components/common/spinner/Spinner';
-import AccountArea from '@/components/SettingPage/AccountArea';
-import LogOutBtn from '@/components/SettingPage/LogOutBtn';
-import ProfileArea from '@/components/SettingPage/ProfileArea';
+import SettingContent from '@/components/SettingPage/SettingContent';
+import SettingMenu from '@/components/SettingPage/SettingMenu';
+
+type TabType = 'account' | 'category' | 'routine' | 'history';
 
 function Setting() {
-	const CALENDAR_CLIENT_ID = import.meta.env.VITE_GOOGLE_CALENDAR_CLIENT_ID;
+	const [activeTab, setActiveTab] = useState<TabType>('account');
+
+	const handleTabChange = (tab: TabType) => {
+		setActiveTab(tab);
+	};
 	const { data: userInfo, isLoading } = useGetUserInfo();
 	if (isLoading) {
 		return <LoadingSpinner />;
 	}
 
 	return (
-		<GoogleOAuthProvider clientId={CALENDAR_CLIENT_ID}>
-			<SettingContainer>
-				<Wrapper>
-					<ProfileArea userData={userInfo?.data} />
-					<AccountArea calendarAccount={userInfo?.data.googleCalenders} />
-				</Wrapper>
-				<LogOutBtn />
-			</SettingContainer>
-		</GoogleOAuthProvider>
+		<SettingLayout>
+			<NavBar isOpen={false} handleSideBar={() => {}} />
+			<SettingArea>
+				<SettingHeader>설정</SettingHeader>
+				<MainWrapper>
+					<SettingMenu activeTab={activeTab} onTabChange={handleTabChange} />
+					<SettingContent activeTab={activeTab} userData={userInfo?.data} />
+				</MainWrapper>
+			</SettingArea>
+		</SettingLayout>
 	);
 }
 
 export default Setting;
 
-const Wrapper = styled.div`
+const SettingLayout = styled.div`
+	display: flex;
+	height: 108rem;
+	overflow: hidden;
+`;
+
+const SettingArea = styled.div`
 	display: flex;
 	flex-direction: column;
-	justify-content: flex-start;
+	align-items: center;
+	width: 182.4rem;
+	height: 106.4rem;
```

Comment:
```suggestion
	width: 100%;
```

마진 줬으니까 100%로 처리하는 게 더 좋을 것 같아요!

Reply:
네!!